crux-mir/src/Mir/Overrides.hs

```diff
@@ -453,6 +477,24 @@ bindFn symOnline cs name cfg
           pfxInit == edidInit &&
           "_inst" `Text.isPrefixOf` edidLast
 
+    showRV :: TypeRepr tp' -> RegValue sym tp' -> String
+    showRV (BVRepr _) x = show (printSymExpr x)
+    showRV AnyRepr (AnyValue tpr' rv') = "(AnyValue " ++ show tpr' ++ " " ++ showRV tpr' rv' ++ ")"
+    showRV UnitRepr () = "Unit"
+    showRV (MaybeRepr tpr') Unassigned = "(Maybe Unassigned)"
```

Comment:
For consistency with the `PE` case, should this print the `TypeRepr`?

Reply:
I went the other direction when adding `MaybeRepr` support to `ppRegVal` and made neither case print the `TypeRepr`.  Most of the other `ppRegVal` cases don't print any `TypeRepr`s, so I figured it was more consistent to leave it out.